doc/introduction/interfaces/jax.rst

```diff
@@ -89,14 +95,14 @@ DeviceArray(-3.4332792e-10, dtype=float32)
 Using jax.jit on QNodes
 -----------------------
 
-To fully utilize the power and speed of JAX, you'll need to just-in-time compile
-your functions - a process called "jitting". 
-If you're only taking expectation values, you can simply add the decorator on your QNode directly.
+To fully utilize the power and speed of JAX, you'll need to just-in-time compile your functions - a
+process called "jitting". If you're only taking expectation values, you can simply add the decorator
+on your QNode directly.

```

Comment:
I thought you don't like the "you" in docs? :)

Reply:
🤔 Let me rephrase this.